tests/lib/rules/array-callback-return.js

```diff
@@ -14,11 +14,22 @@ const rule = require("../../../lib/rules/array-callback-return"),
 
 const ruleTester = new RuleTester();
 
+const options = [{ allowImplicit: true }];
+
 ruleTester.run("array-callback-return", rule, {
     valid: [
+
+        // options: { allowImplicit: false }
```

Comment:
I'd love to see at least one test that explicitly tests with `{ allowImplicit: false }` as a safeguard against regressions and to make sure we're exercising all code paths.